_config.yml

```diff
@@ -70,6 +70,13 @@ aux_links:
 # Makes Aux links open in a new tab. Default is false
 aux_links_new_tab: false
 
+# Dropdown settings
+drop_title: "Dropdown"
+
+drop_links:
+  "Link":
+    - "//github.com/pmarsceill/just-the-docs"
```

Comment:
Could we start these with `https://`?

Reply:
This was copied and pasted from the aux link code just above and will be overwritten through site anyways so I think it's fine.